### PR TITLE
feat: improve release steps

### PR DESCRIPTION
**Pull request by sravan-s**
* Update readme
* Auto add version number to dist/index.d.ts
* Generate package.json in dist automatically
* Cleanup PR template

fixes: https://sendbird.atlassian.net/browse/UIKIT-3471

### CODE REVIEW

File: scripts/steps.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+4. Update the version in `package.json` and `package-lock.json` to the new Release Candidate version: Run `npm version prepatch/preminor/premajor`
+5. Run `npm run build`
+6. Inside `./<root>/dist`, Run `npm publish --tag beta` for beta release. You can also use `npm publish --tag <next | rc>` for other release candidates
+7. Commit .bundle_analysis.json, encouraged to summarize the bundle-size in the changelog
+8. Test the rc version in your project(s) https://github.com/sendbird/uikit-react-test_suite (WIP: We are automating this step)
+9. If everything is fine go to Step 2, otherwise make changes to main branch, rebase and repeat Step 1
+
+## Step 2 - Actual release
+10. Commit all changes
+11. Update the version in `package.json` and `package-lock.json` to the new Release Candidate version: Run `npm version patch/minor/major`
```
Comment (HoonBaek) [inline scripts/steps.md:16]: typo: (real) release version

Comment (sravan-s) [reply scripts/steps.md:16]: good catch

**Review by HoonBaek — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -9,20 +9,9 @@ and we'll evaluate if we can setup a [CLA](https://en.wikipedia.org/wiki/Contrib
 
 ## For Internal Contributors
```
Comment (HoonBaek) [inline .github/pull_request_template.md:10]: Don't we remove this? because we move the ticket link to the bottom

Comment (sravan-s) [reply .github/pull_request_template.md:10]: its okay to be here, no? :) I changed title precedence for `## Description Of Changes ` to `### Description Of Changes `

**Pull request merged**